app/controllers/collections_controller.rb

```diff
@@ -36,6 +35,19 @@ def create
     end
   end
 
+  def create_ajax
+    @collection = Collection.new(collection_params)
+    @collection.users << current_user
+    authorize @collection
+
+    if @collection.save
+      redirect_to @collection.tasks.first, notice: t('.success_notice')
```

Comment:
Is a redirect the expected behavior for an AJAX call? 🤔 I'd expect a JSON response, no?